### PR TITLE
Bug 858976 - cmd+click a link in a panel should open the link in a new tab

### PR DESCRIPTION
**Pull request by ZER0**

### CODE REVIEW

File: lib/sdk/panel/utils.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ const { getMostRecentBrowserWindow, getOwnerBrowserWindow,
 const { create: createFrame, swapFrameLoaders } = require("../frame/utils");
 const { window: addonWindow } = require("../addon/window");
 const { isNil } = require("../lang/type");
+const { openTab } = require("../tabs/utils");
```
Comment (Gozala) [inline lib/sdk/panel/utils.js]: This does not seems to be used, please loose it.

File: lib/sdk/panel/utils.js [hunk 2]
```diff
@@ -295,11 +307,27 @@ function make(document) {
     events.emit(type, { subject: panel })
   }
 
+  function onPanelClick(event) {
```
Comment (Gozala) [inline lib/sdk/panel/utils.js]: This does not captures anything from the scope so please this function top level instead.

**Comment by benmccann** [comment]
@ZER0 would you be able to update this PR to fix the merge conflict?

**Comment by benmccann** [comment]
@ZER0 could you fix the merge conflict?

**Comment by ZER0** [comment]
@benmccann my apologies, I totally didn't see the github notifications about that, I definitely need to unify bugzilla and github notifications somehow! I just update the PR, and after travis is done I'm going to merge the patch.

**Comment by benmccann** [comment]
Thanks!

**Pull request merged**